bookmarklet.py

```diff
@@ -31,15 +33,8 @@
 app.config['SESSION_TYPE'] = 'filesystem'
 Session(app)
 
-def get_event_id(url):
-    url_components = urlparse(url)
-    if not url_components.netloc == 'www.eventbrite.com':
-        return None
-
-    # hackily parsing eventbrite urls
-    event_name = url_components.path.split('/')[-1]
-    event_id = event_name.split('-')[-1]
-    return event_id
+URL_HANDLERS = {'www.eventbrite.com' : eventbrite_saver.EventbriteSaver,
```

Comment:
What if the www prefix is not there?

Reply:
I don't think that can happen if you're using the tool correctly. Remember that it's meant to be used as a bookmarklet -- the user will be clicking our bookmarklet from an already-loaded eventbrite page, which always resolves to a www.eventbrite page even if you type in a url that doesn't contain it. So this is what urlparse will return for the netloc.